src/instrumentations/extentions/aws/SQSEventCreator.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright The Cisco Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Span } from '@opentelemetry/api';
+import {
+  AwsSdkRequestHookInformation,
+  AwsSdkResponseHookInformation,
+} from '@opentelemetry/instrumentation-aws-sdk';
+import { AWSEventCreator } from './event-creator-interface';
+
+export class SQSEventCreator implements AWSEventCreator {
+  requestHandler(span: Span, requestInfo: AwsSdkRequestHookInformation): void {
+    const queueUrl = requestInfo.request.commandInput.QueueUrl;
+    const splited = queueUrl.split('/');
+    span.setAttribute('aws.sqs.queue_name', splited.pop());
```

Comment:
are this always returns array of the experted field values?

Reply:
I'll fix this